httpx/utils.py

```diff
@@ -107,6 +107,16 @@ def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
     return netrc_info.authenticators(host)  # type: ignore
 
 
+def get_ssl_cert_file() -> typing.Optional[str]:
```

Comment:
Let's change the title of this since "cert" is what we use for client cert. Really this should be `get_ca_bundle_from_env()` or something like it?